src/autoComplete/index.tsx

```diff
@@ -90,6 +96,12 @@ export default class AutoComplete extends Component<AutoCompleteProps, AutoCompl
     defaultValue: '',
     showList: true,
     valid: false,
+    autoCorrect: 'off',
```

Comment:
Wouldn't it be better to directly extend the Textfield props? Or are there some props you don't want to have

Reply:
Do you mean those?
```
export interface CommonFormFields {
  name: string,
  id?: string,
  type?: 'text' | 'password' | 'email' | 'number' | 'search',
  placeholder?: string,
  maxLength?: number,
  autoCorrect?: 'on' | 'off',
  autoComplete?: 'on' | 'off',
  disabled?: boolean,
  readOnly?: boolean,
  autoFocus?: boolean,
  required?: boolean,
  onFocus?: (event:
    React.FocusEvent<HTMLTextAreaElement> | React.FocusEvent<HTMLInputElement>) => void,
  onBlur?: (event:
    React.FocusEvent<HTMLTextAreaElement> | React.FocusEvent<HTMLInputElement>) => void,
}
```